content/blog/fish-extract.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "`extract` in fish"
+description = ""
+tags = ["development", "fish", "shell"]
+categories = ["Development"]
+# series = []
+date = 2019-11-14T10:40:55-05:00
+draft = false
++++
+I recently switched from `zsh` to `fish` as my shell of choice.
+I liked the idea of starting from scratch, with the sane defaults that `fish` provides, as my [`zsh` configuration files were getting a bit out of control](https://github.com/svanburen/dotfiles/blob/master/zshrc).
```

Comment:
**TYPO - "SANE" SHOULD BE "SAME"

Reply:
"sane" is actually right here, given that I'm referring to the (somewhat) "insane" defaults of other shells, heh